tests/lib/rules/jsx-indent.js

```diff
@@ -159,6 +188,15 @@ ruleTester.run('jsx-indent', rule, {
       '  );',
       '}'
     ].join('\n'),
+    output: [
+      'function App() {',
+      '  return (',
+      '    <App>',
+      '  <Foo />',
+      '</App>',
+      '  );',
+      '}'
+    ].join('\n'),
```

Comment:
This is an error test case, right? If I'm reading it right, it's saying that `</App>` should be indented by 4 spaces but is only indented by 0.